llms/mlx_lm/generate.py

```diff
@@ -3,6 +3,7 @@
 
 import mlx.core as mx
 
+from .models import plamo
```

Comment:
nit: unused import :)

Reply:
Thanks! removed in this commit: https://github.com/ml-explore/mlx-examples/pull/303/commits/d213195ee8c55861a904d92256461dea6aad5abf